2022-01-27 07:48:04,145 INFO    MainThread:4405 [wandb_setup.py:_flush():71] setting env: {}
2022-01-27 07:48:04,146 INFO    MainThread:4405 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-27 07:48:04,146 INFO    MainThread:4405 [wandb_init.py:_log_setup():371] Logging user logs to /home/sohui/code/DTC/code/wandb/run-20220127_074804-r77i5dyp/logs/debug.log
2022-01-27 07:48:04,146 INFO    MainThread:4405 [wandb_init.py:_log_setup():372] Logging internal logs to /home/sohui/code/DTC/code/wandb/run-20220127_074804-r77i5dyp/logs/debug-internal.log
2022-01-27 07:48:04,147 INFO    MainThread:4405 [wandb_init.py:init():404] calling init triggers
2022-01-27 07:48:04,147 INFO    MainThread:4405 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'mdoel': 'DTC', 'server': 57, 'image size': '112,112,80'}
2022-01-27 07:48:04,147 INFO    MainThread:4405 [wandb_init.py:init():449] starting backend
2022-01-27 07:48:04,148 INFO    MainThread:4405 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-27 07:48:04,165 INFO    MainThread:4405 [backend.py:ensure_launched():199] starting backend process...
2022-01-27 07:48:04,174 INFO    MainThread:4405 [backend.py:ensure_launched():204] started backend process with pid: 4485
2022-01-27 07:48:04,176 INFO    MainThread:4405 [wandb_init.py:init():458] backend started and connected
2022-01-27 07:48:04,178 INFO    MainThread:4405 [wandb_init.py:init():519] updated telemetry
2022-01-27 07:48:04,180 INFO    MainThread:4405 [wandb_init.py:init():549] communicating current version
2022-01-27 07:48:05,071 INFO    MainThread:4405 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-27 07:48:05,072 INFO    MainThread:4405 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-01-27 07:48:05,328 INFO    MainThread:4405 [wandb_init.py:init():592] starting run threads in backend
2022-01-27 07:48:05,628 INFO    MainThread:4405 [wandb_run.py:_console_start():1816] atexit reg
2022-01-27 07:48:05,630 INFO    MainThread:4405 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-01-27 07:48:05,630 INFO    MainThread:4405 [wandb_run.py:_redirect():1695] Redirecting console.
2022-01-27 07:48:05,637 INFO    MainThread:4405 [wandb_run.py:_redirect():1751] Redirects installed.
2022-01-27 07:48:05,637 INFO    MainThread:4405 [wandb_init.py:init():619] run started, returning control to user process
2022-01-27 07:48:10,022 INFO    MainThread:4405 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-01-27 07:48:10,023 INFO    MainThread:4405 [wandb_run.py:_restore():1758] restore
2022-01-27 07:48:12,243 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 596
  total_bytes: 596
}

2022-01-27 07:48:12,450 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 596
  total_bytes: 596
}

2022-01-27 07:48:12,706 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 596
  total_bytes: 596
}

2022-01-27 07:48:13,365 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 596
  total_bytes: 5552
}

2022-01-27 07:48:13,470 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 596
  total_bytes: 5578
}

2022-01-27 07:48:13,573 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 596
  total_bytes: 5578
}

2022-01-27 07:48:13,678 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1024
  total_bytes: 5578
}

2022-01-27 07:48:13,781 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5578
  total_bytes: 5578
}

2022-01-27 07:48:13,885 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5578
  total_bytes: 5578
}

2022-01-27 07:48:13,988 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5578
  total_bytes: 5578
}

2022-01-27 07:48:14,092 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5578
  total_bytes: 5578
}

2022-01-27 07:48:14,388 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5578
  total_bytes: 5578
}

2022-01-27 07:48:14,913 INFO    MainThread:4405 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5578
  total_bytes: 5578
}
local_info {
}

2022-01-27 07:48:16,069 INFO    MainThread:4405 [wandb_run.py:_append_files():2184] logging synced files
